src/biogeochem/ch4Mod.F90

```diff
@@ -3453,14 +3453,14 @@ logical                , intent(in)    :: lake      ! function called with lake
                  ch4_aere_depth(c,j) - ch4_ebul_depth(c,j) ! [mol/m3-total/s]
             ! aerenchyma added to surface flux below
             ! ebul added to soil depth just above WT
-            if (source(c,j,1) + conc_ch4(c,j) / dtime < -1.e-12_r8) then
```

Comment:
Can this error check go back to what it was?

Reply:
I didn't notice this change before. I have changed it back and we need further test.